### DISCUSSION TITLE
What is the only country in the world that is both an enclave and an exclave?

### DISCUSSION THREAD

**Discussion by nsafarova** [ParanoidUser/yolo#639] [Galaxy Trivia] [answered]
**Bonus**: What is the smallest country in the world by land area?

**Comment by peterbaccaro** [comment]
**OMAN** in UAE is an exclave and an enclave. It's an exclave because it's a part of one territory surrounded by another. It also happens to be an enclave because it is surrounded by only one entity.

The smallest country in the world is **VATICAN CITY**, with a landmass of just 0.49 square kilometers (0.19 square miles).

**Comment by ParanoidUser** [comment, marked as answer]
Great questions, **Nigar**! 👏👏 Thank you for your active participation in the project. Keep them coming! 

---
Yup, it's gonna be **OMAN**. Sometimes unofficially referred to as the UAE Donut🍩 due to this country geographical peculiarity. In fact, it's enclaved and exclaved by the same country, the UAE.

<img width="299" alt="image" src="https://user-images.githubusercontent.com/5120290/235010640-c648f7d9-da1d-4fc6-90b5-604e216f4152.png">

Did you know that despite its modest size, **Vatican City** issues its own passports! The next smallest would be Monaco, which is 4 times larger than Vatican.

<img width="252" alt="image" src="https://user-images.githubusercontent.com/5120290/235011476-97d80aa5-22f8-48e3-af22-e38616d993e2.png">
